weblate/vcs/base.py

```diff
@@ -200,6 +201,8 @@ def _popen(
             raise RepositoryException(
                 process.returncode, process.stdout + (process.stderr or "")
             )
+        if raw:
+            return BytesIO(process.stdout)
```

Comment:
What is the reason for wrapping in `BytesIO`? This can easily break existing consumers. It should always return the content and the caller would wrap it in `BytesIO` when needed.

Reply:
The reason is `respository.get_file(` gives error if binary content is sent by this method when I get the file to save it.

Reply:
I understand that adding `raw` to `get_file` is needed. What I'm questioning is wrapping the content in `BytesIO` as that changes existing behavior without a good reason.

Reply:
After reading the file from git checkout, we don't need this anymore so I will remove it.